File: src/routes/health.rs
// Copyright 2025 AtomArtist. All rights reserved.

use axum::Json;
use serde_json::{json, Value};

pub async fn health_check() -> Json<Value> {
    Json(json!({
        "status": "healthy",
        "service": "atomartist"
    }))
}

File: src/routes/health.py
# Copyright 2025 AtomArtist. All rights reserved.

from flask import Blueprint, jsonify

health_bp = Blueprint('health', __name__)


@health_bp.route('/health')
def health_check():
    return jsonify({
        'status': 'healthy',
        'service': 'atomartist'
    })

